### DISCUSSION TITLE
Dashboard settings: parameter for setting default view mode

### DISCUSSION THREAD

**Discussion by vdwpsmt** [grafana/grafana#46566] [Dashboards, Panels & Visualizations]
use case: using the **dashboardlist** panel for navigation it is not possible to open a dashboard in other then default view mode.  Suppose we want to use all our dashboards in tv mode, with a navigation or home page using the default dashboardlist panel this does not seem possible.  
My suggestion would be to have a general dashboard setting to set the default view mode when calling a dashboard without any extra url parameters.
Any thoughts about that?  workarounds?

**Comment by torkelo** [comment]
another option would be to preserve kiosk/tv mode when navigating between dashboards

**Reply by vdwpsmt** [reply]
ok, in that way it is not really a property of the dashboard itself?  Are there similarities with the possibility to set the default time range for a dashboard?

**Reply by asimakos** [reply]
@vdwpsmt Sorry for the interruption! Is it possible to share the following https://github.com/grafana/grafana/issues/9310 ? It would help me a lot! Regards 🙏

**Comment by torkelo** [comment]
not really, well today the view mode is not persisted at all

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
